internal/wrappers/features/step_defs_cpp/3rdparty/json_spirit/json_spirit.vcproj

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="Windows-1252"?>

```

Comment:
I don't think this should be commited :p

Reply:
It's a file that comes with the third party library

Reply:
This file is part of the 3rd party lib...